backend/dal/Migrations/v1.0.14.0/Up/PostUp/01-TableDefinition.sql

```diff
@@ -0,0 +1,231 @@
+DELETE FROM "dbo"."PIMSX_TableDefinitions"
```

Comment:
Rather then modifying this table piecemeal to get the desired state, since this table is out of sync, I've just overridden the existing data with an exact copy of the desired state. Going forward this table will be kept in sync.